### ISSUE TITLE
Neve FSE Landing Page Themeisle

### ISSUE THREAD

**Issue by preda-bogdan** [Codeinwp/neve-fse#24]
To prepare for the launch we need to also have the Neve FSE landing page ready on the Themeisle website.

I can copy one of the pages from our other FSE themes as a base, maybe Raft.

@Codeinwp/design-team Can you help with some guidance here?

@harshitarora-in I'm adding you to the loop for the copyright part.

Let me know your input we can have everything ready by the end of next week I think.

**Comment by mghenciu** [comment]
A good start would be to clone [Raft page](https://themeisle.com/themes/raft/), @preda-bogdan; and from there we can all colaborate there, design team with screenshots and layout, Harshit with text and you with implementing more complex things.

In terms of Content @harshitarora-in , it will probably be pretty much the same ideas as we have for other FSE themes; the difference here will be presenting how we adapted some of the features from Neve to Neve FSE, basically letting users know the 'soul' of Neve is still here :)

**Comment by preda-bogdan** [comment]
@JohnPixle @mghenciu and @harshitarora-in I've added a page here https://fmbklha76y-staging.onrocket.site/themes/neve-fse/ that is a duplicate of the Raft page that we can use before adding to production.

**Comment by harshitarora-in** [comment]
>In terms of Content @harshitarora-in , it will probably be pretty much the same ideas as we have for other FSE themes; the difference here will be presenting how we adapted some of the features from Neve to Neve FSE, basically letting users know the 'soul' of Neve is still here :)

Thanks for the heads up, I will start working on the copywriting part and will notify here once I am done!

**Comment by harshitarora-in** [comment]
@JohnPixle Yes I made some text edits yesterday and you are right I was having some issues with the font so I replaced the font with SF Pro, is that okay?

**Comment by harshitarora-in** [comment]
I have edited the content as well as one screenshot in the Figma design, please do let me know your feedback. 

Thank you!

**Comment by preda-bogdan** [comment]
@harshitarora-in
cc: @JohnPixle 
Is this ready to be added to staging and live? Are there any changes required here?

**Comment by mghenciu** [comment]
@preda-bogdan , just a quick question, when you have some time.
We were discussing with Harshit about the text on the Landing Page, and wanted to ask if with this line:

### Easy translation
_Elements in this theme can be translated with free and premium plugins. Plus, you can use any RTL language._
<img width="1361" alt="Screenshot 2023-05-04 at 13 00 19" src="https://user-images.githubusercontent.com/52494172/236173625-1dcf985c-a98a-48ae-b98d-3139109ff483.png">


**we are not overpromising something**? I mean that the theme will work with Translation and RTL plugins. Can you please confirm?

**Comment by preda-bogdan** [comment]
@mghenciu Yes, we are using the i18n translation functions for the texts as it is required for any theme and plugin. However there are some texts inside the `html` files that are not translatable. But since they are part of the content it should be fine as plugins that allow content translation would work as expected.

Let me know if this clears things up. In my opinion it should be fine.
Thank you!

**Comment by preda-bogdan** [comment]
@Codeinwp/design-team 
cc; @harshitarora-in 
I've included the design and text changes here: https://fmbklha76y-staging.onrocket.site/themes/neve-fse/

Let me know if you have any feedback.
Thank you!

**Comment by preda-bogdan** [comment]
I've updated the spacing and fixed the alignment issue, Thank you for mentioning it.

**Comment by mghenciu** [comment]
Looking good, Bogdan, top work.
The only thing I noticed (and I think it was caused by Figma), is the border radius in the last image: 
<img width="1668" alt="Screenshot 2023-05-04 at 19 37 30" src="https://user-images.githubusercontent.com/52494172/236269082-b187a020-5e6a-4b67-9e7f-6d6fdf22af86.png">

meaning that all images have border radius, but the one **on the right side is cut**. 

I've fixed it in Figma. Also fixed the other image from there that had some strange border.
**Just re-exporting and replacing the image on the Landing Page - should fix the issues.**

**Comment by preda-bogdan** [comment]
@mghenciu Thank you, I've updated the image.

**Comment by preda-bogdan** [comment]
@JohnPixle I've already did this for staging https://staging.demosites.io/neve-fse/ and will replicate it on live just for this purpose.

**Comment by rodica-andronache** [comment]
@preda-bogdan not sure if you've already saw this, but on mobile, there is some big spacing in this section https://vertis.d.pr/i/fo3fGX and the content here https://vertis.d.pr/i/Yq0OER does not have left/right space

**Comment by preda-bogdan** [comment]
@rodica-andronache Thank you for mentioning this, I've solved it on live. I am awaiting for some changes to be merged to Themeisle and will publish the page. 👍🏼
